libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -272,14 +272,14 @@ void WaveTrack::Interval::SetTrimRight(double t)
       GetClip(channel)->SetTrimRight(t);
 }
 
-void WaveClip::Interval::TrimLeft(double deltaTime)
+void WaveTrack::Interval::TrimLeft(double deltaTime)
```

Comment:
... oh, there we go. You might want to squash these with the previous commit, which probably won't compile.

Reply:
Yes